flood_adapt/object_model/direct_impacts.py

```diff
@@ -367,8 +396,8 @@ def _create_aggregation(self, metrics_path):
                 file_format="geopackage",
             )
 
-    def _create_footprints(self, fiat_results_path):
-        logging.info("Create footprints...")
+    def _create_footprints(self, results):
```

Comment:
Also here with just using results. Something like fiat_results would be more descriptive